kpatch-build/kpatch-build

```diff
@@ -1238,6 +1238,10 @@ remove_patches
 
 cp -LR "$DATADIR/patch" "$TEMPDIR" || die
 
+# cp preserves mode and the files might have been read-only. This would
+# interfere with cleanup later.
+chmod -R u+rw "$TEMPDIR" || die
+
```

Comment:
The only incredibly small nit I have is that we could group this together like so:
``` bash
# cp preserves mode and the files might have been read-only. This would
# interfere with cleanup later, so ensure the $TEMPDIR is read/write.
cp -LR "$DATADIR/patch" "$TEMPDIR" || die
chmod -R u+rw "$TEMPDIR" || die
```

Reply:
Fixed!